src/backend/src/modules/pia-intake/jsonb-classes/collection-use-and-disclosure/steps-walkthrough.class.ts

```diff
@@ -0,0 +1,44 @@
+import { IsString } from '@nestjs/class-validator';
+import { UserTypesEnum } from 'src/common/enums/users.enum';
+import { IFormField } from 'src/common/interfaces/form-field.interface';
+
+export class StepWalkthrough {
+  @IsString()
+  drafterInput?: string;
+
+  @IsString()
+  mpoInput?: string;
+
+  @IsString()
+  foippaInput?: string;
+
+  @IsString()
+  OtherInput?: string;
+}
+
+export const stepWalkthroughDetails: Array<IFormField<StepWalkthrough>> = [
+  {
+    key: 'drafterInput',
+    type: 'text',
+    isRichText: false,
+    allowedUserTypesEdit: null, // any
```

Comment:
not understanding the comments  '// any' what does it mean?

Reply:
the idea is any role in the system can access and edit this field. will place restrictions soon in the json when we will use this field.

Reply:
updated to a better comment.